docs/learn/using-zeitgeist-markets.md

```diff
@@ -300,49 +171,97 @@ dispute for the token which the market eventually resolved to receive their
+$x \cdot y = \mathrm{const}$ formula which allows different assets to have
+different _weights_, which define their impact on price.
 
-Zeitgeist's AMM, the Rikiddo scoring rule, does not guarantee that for any
-market the sum of the prices of all outcome assets is approximately equal to 1
-ZTG, or that the price of any outcome tokens remains below 1 ZTG. High prices
-are indicators of low liquidity or a volatile market.
+This AMM's liquidity pool contains balances of ZTG and of all outcome tokens of
+the market. Users can trade their tokens with tokens stored in the pool: They
+can buy outcome tokens from the pool with ZTG which is then added to the pool,
+or sell outcome tokens to the pool for some of the pool's ZTG.
 
-This creates opportunities for users to add or remove liquidity to profit from
-arbitrage:
+By default, a new market has no liquidity pool. Instead, the pool must either be
+deployed by the market creator, or by some external liquidity provider. After
+the pool is created, others may _join_ the liquidity pool by providing
+additional liquidity. When deploying liquidity into a pool, a liquidity provider
+will usually provide the same amount of full sets of outcome tokens as ZTG ($x$
+of each outcome token and $x$ ZTG). The current minimum for $x$ is 100, making a
+total value of 200 ZTG.
 
-- If the sum of a market's prices is greater than 1 ZTG, then a user can mint a
-  full set for 1 ZTG and sell it to the liquidity pool at a price above 1 ZTG
-  (ignoring slippage and trading fees). This move will add liquidity to the
-  market.
-- If the sum of a market's prices is less than 1 ZTG, then a user can buy one of
-  each outcome token from the market for less than 1 ZTG and then burn the full
-  set to receive 1 ZTG from the prize pool. This move will remove liquidity from
-  the market.
+For example, lets say the JWST market has no liquidity pool yet and Alice wishes
+to deploy a pool. First she mints 100 full sets of outcome tokens, so she pays
+100 ZTG into the prize pool of the market and receives 100 JWSTYES and 100
+JWSTNO. Then she transfers these outcome tokens plus 100 ZTG into the pool. The
+whole endeavor costs her 200 ZTG plus transaction costs and earns her 100
+liquidity shares.
 
-### Liquidity Pools for Multiple Markets
+Suppose now that the market ends and the balances of the pool are the following:
+63 JWSTYES, 89 JWSTNO, and 120 ZTG. The balance of ZTG has increased from
+trading fees. After market close, Alice withdraws her funds: The outcome tokens
+and 120 ZTG. If the JWST did not launch on December 18, then she can redeem the
+89 JWSTNO for 89 ZTG from the prize pool. This means that she's made a gain of 9
+ZTG for supplying liquidity to the pool. If, on the other hand, the JWST does
+launch December 18, Alice is left holding 120 ZTG and 63 JWSTYES (redeemable for
+63 ZTG), and, thus, Alice got rekt to the tune of 17 ZTG (but many traders will
+have made some profit).
```

Comment:
```suggestion
63 ZTG). As a result, Alice will incur a net loss of 17 ZTG, while many traders might realize a profit.
```

Reply:
We should avoid colloquial language on our formal and professional documentation site.